Cargo.toml

```diff
@@ -17,11 +17,11 @@ rand_core = "0.5"
 lazy_static = "1.4.0"
 
 [features]
-default = []
 derive = ["fff_derive"]
+default = ["derive"]
 
 [badges]
 maintenance = { status = "actively-developed" }
 
-[target.'cfg(target_arch = "x86_64")'.build-dependencies]
+[build-dependencies]
```

Comment:
please revert, this will trigger issues when building for things like `wasm`

Reply:
Done with 02b5f8d